examples/set-annotations/advanced/README.md

```diff
@@ -1,14 +1,31 @@
 # set-annotations: Advanced Example
 
-The `set-annotation` function adds annotations to KRM resources.
+### Overview
 
-We use the following `SetAnnotationConfig` to configure the function.
+This example demonstrates how to declaratively run [`set-annotations`] function
+to upsert annotations to the `.metadata.annotations` field on all resources.
+
+We use the following `Kptfile` and `fn-config.yaml` to configure the function.
+
+```yaml
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-annotations:unstable
+      configPath: fn-config.yaml
+```
 
 ```yaml
+# fn-config.yaml
 apiVersion: fn.kpt.dev/v1alpha1
 kind: SetAnnotationConfig
 metadata:
-  ...
+  name: my-func-config
+  annotations:
+    config.kubernetes.io/local-config: 'true'
```

Comment:
Is this needed in kpt v1 ?

Reply:
In kpt v1, kpt live still understand it.
I want to ensure the example package is compatible with `kpt live` (IOW the package should be able to kpt live apply after the transformation).
Otherwise, the fn config CRD will be submitted to the server and cause an error.

Reply:
Shouldn't kpt live also behave the same way as fn and identify such configs pointed from Kptfile ?

@frankfarzan

Reply:
+1

Reply:
We shouldn't need `local-config` as live should understand the semantics. There's a tracking issue for this and @mortent is driving it in m3.